internal/testutil/mig/provider.go

```diff
@@ -9,6 +9,13 @@ import (
 	"github.com/mongodb/terraform-provider-mongodbatlas/internal/testutil/acc"
 )
 
+func SkipIfVersionBelow(t *testing.T, minVersion string) {
```

Comment:
better tb *testing.TB as the other methods

Reply:
interestingly I got a linting error in my precommit hook, locally I have the same version as our CI job (golangci-lint has version 1.55.0)

> internal/testutil/mig/provider.go:12:6: parameter *testing.T should have name t (thelper)

Reply:
what's the linter error? we're using tb in many methods with no issues

Reply:
notice that you have to say *testing.TB not *testing.T

Reply:
B is for Benchmark, in case we want to have those tests at some moment

Reply:
thanks for clarifying, did not notice that detail. Adjusted for convention.